tests/rustdoc-json/primitives/no_primitive_impl_pollution.rs

```diff
@@ -0,0 +1,2 @@
+//@ count "$.index[*]" 1
```

Comment:
It'd be nice if this could link back to the original issue, and also add a comment of *what* it's testing. At the moment, the purpose of this test won't be clear when viewed without context.

Reply:
I also renamed the test to `the_smallest.rs`, but I'm not sure if that's the most fitting name